src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -621,6 +634,25 @@ private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
             this.progressIndicator?.Stop();
             Output.WriteLine(string.Empty, OutputLevel.Information);
 
+            if (verbosityLevel == Verbosity.Quiet)
+            {
+                foreach (var sd in SummaryDictionary.ToArray())
+                {
+                    var summary = SummaryDictionary[sd.Key];
+
+                    // Failed! Pass {1} Failed {2} skipped {3} Time : 233 se ({4})
+                    if (summary.FailedTests > 0)
+                    {
+                        Output.Information(false, ConsoleColor.Red, string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryFailed, summary.TotalTests, summary.PassedTests, summary.FailedTests, summary.SkippedTests, GetFormattedDurationString(summary.TimeSpan), sd.Key.Split('\\').Last()));
```

Comment:
>Information [](start = 31, length = 11)

Can you also attach a screenshot of the summary?

Reply:
added